grammar/rainerscript.c

```diff
@@ -2374,66 +2447,16 @@ doFuncCall(struct cnffunc *__restrict__ const func, struct svar *__restrict__ co
 		cnfexprEval(func->expr[0], &r[0], usrptr, pWti);
 		str = (char*) var2CString(&r[0], &bMustFree);
 
+		bMustFree2 = 0;
```

Comment:
This smells strongly like a constant expression... (see next comment)

Reply:
Just initializing it, because it is conditionally set in the following `if ()`. I think one of the Solaris build systems was giving a warning because `bMustFree2` "contains garbage".

Reply:
yeah ... but have you read the other comment below? I think the problem is not that it is initialized, but that it is queried albeit it is not used.

Let me re-phrase the questions: what is ```fmt``` and when does it actually need to be free'ed? Maybe it's even a logic bug.

Reply:
Yes, I read it ;)

`fmt` is the optional second parameter to `is_time()`. If `is_time()` is called without the format specifier, then `fmt` won't have to be freed. If it is specified, it has to be converted to a cstr, and freed after. No?

Maybe I should have mentioned the usage in the pull request details. I can do that later when I have access to an actual keyboard (in a few hours).

Reply:
let me check... I did not see whre bMustFree2 was written except in the initializer. Maybe gotten lost in diff... (and, yeah, I know that with the real keyboards -- you are doing extremely well ;-)).

Reply:
damn, my bad -- I really got lost in the diff. Overlooked the place where bMustFree2 is written...